cmd/janitor/gcp_janitor.py

```diff
@@ -147,6 +147,7 @@
     'us-central1-b',
     'us-central1-c',
     'us-central1-f',
+    'us-east1-a',
```

Comment:
`us-east1-a` probably didn't exist when this list was populated.

Are there any other zones that should be added? See the comment on L93 describing how this list was generated.

Reply:
Hmm I tried running the command on L93, it did generate a few more zones but did not include `us-east1-a`. 

After I added `us-east1-a` in this list, I did try running the script against a project that has GCE instances in the `us-east1-a` zone, and it worked.

Reply:
Per offline discussion, us-east1-a is a private zone and is not exposed to all projects, so removing it and update the `ZONES` list with the gcloud command instead.